### ISSUE TITLE
`emmylua_check` should report parsing errors in config files.

### ISSUE THREAD

**Issue by lewis6991** [EmmyLuaLs/emmylua-analyzer-rust#512]
My `.emmyrc.json` had a incorrect `,` and was ignored.

Would be much more helpful if `emmylua_check` reports when it cannot correctly read a config file.

```lua
{
  "diagnostics" : {
    "disable" : [
      "unnecessary-if"
    ]
  },
  "strict": {
    "typeCall": true,
    "arrayIndex": true
  },
}
```

Perhaps the log module should direct to warnings and errors to `stdout` for `emmylua_check.
